demo/angular/gridstack-item.component.ts

```diff
@@ -0,0 +1,34 @@
+import {ChangeDetectionStrategy, Component, ElementRef, Input, NgZone, OnInit, Renderer2} from '@angular/core';
+
+@Component({
+  selector: 'ef-gridstack-item',
+  templateUrl: './gridstack-item.component.html',
+  styleUrls: ['./gridstack-item.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class GridstackItemComponent implements OnInit {
+
+  @Input() public x: number;
+  @Input() public y: number;
+  @Input() public width: number;
```

Comment:
I intentionally did not use width/height to conflict with DOM existing attributes...

Reply:
It's not an issue. Using `[height]="42"` won't collide with the native attributes. If the user wants to set native height attribute, he would do `[attr.height]="42"`